domain/global_vars_cache_test.go

```diff
@@ -14,7 +14,7 @@
 package domain
 
 import (
-	"sync/atomic"
+	"github.com/pingcap/tidb/util/stmtsummary"
```

Comment:
Please re-arrange the order the imported package

Reply:
Addressed.